00-introduction.md

```diff
@@ -23,6 +23,62 @@ This is version 0.
 10. [BOLT #10](10-dns-bootstrap.md): DNS Bootstrap and Assisted Node Location
 11. [BOLT #11](11-payment-encoding.md): Invoice Protocol for Lightning Payments
 
+## The Spark: A Short Introduction to Lightning
+
+Lightning is a protocol for fast payments using Bitcoin, using a
+network of channels.
+
+### Channels
+
+Lightning works by establishing *channels*: two participants share
+custody of some bitcoin (e.g. 0.1 bitcoin), spendable only with both
+their signatures.
```

Comment:
I feel like this doesn't fully describe what a channel actually is. Maybe: "two participants create a Lightning payment channel that contains some amount of bitcoin (e.g., 0.1 bitcoin) that they've locked up on the Bitcoin network. It is spendable only with both their signatures."

Reply:
Yes, I like that.